helm-chart/crds/retrytimeoutconfig.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: retrytimeoutconfig.nsm.nginx.com
+  labels:
+    app.kubernetes.io/part-of: nginx-service-mesh
+spec:
+  group: specs.smi.nginx.com
+  scope: Namespaced
+  names:
+    kind: RetryTimeoutConfig
+    listKind: RetryTimeoutConfigList
+    shortNames:
+      - rtc
+    singular: retrytimeoutconfig
+    plural: retrytimeoutconfigs
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
```

Comment:
I don't even know if it's necessary, but I've noticed CRDs tend to have `apiVersion`, `kind`, and `metadata` in their list of properties. See meshconfig.yaml as an example.